tests/MongoDB.Driver.Tests/Linq/Linq3ImplementationTests/Serializers/KnownSerializers/KnownSerializersTests.cs

```diff
@@ -0,0 +1,136 @@
+
+        public enum E { A, B };
+
+        private class C1
+        {
+            public E Ei { get; set; }
+        }
+
+        private class C2
+        {
+            public E Ei { get; set; }
+            [BsonRepresentation(BsonType.String)]
+            public E Es { get; set; }
+        }
+
+        private class C3
+        {
+            public E Es { get; set; }
+        }
+
+        private class Results
+        {
+            public bool Result { get; set; }
+        }
+
+        [Theory]
+        [InlineData(E.A, "{ \"Result\" : { \"$eq\" : [ \"$Ei\", 0 ] }, \"_id\" : 0 }")]
+        [InlineData(E.B, "{ \"Result\" : { \"$eq\" : [ \"$Ei\", 1 ] }, \"_id\" : 0 }")]
+        public void Where_operator_equal_should_render_correctly(E value, string expectedProjection)
+        {
+            var subject = GetSubject<C1>();
+
+            var queryable = subject.Select(x => new Results { Result = x.Ei == value });
+
+            AssertProjection<C1, Results>(queryable, expectedProjection);
+        }
+
+        [Theory]
+        [InlineData(E.A, "{ \"Result\" : { \"$eq\" : [ \"$Es\", \"A\" ] }, \"_id\" : 0 }")]
+        [InlineData(E.B, "{ \"Result\" : { \"$eq\" : [ \"$Es\", \"B\" ] }, \"_id\" : 0 }")]
+        public void Where_operator_equal_should_render_enum_as_string(E value, string expectedProjection)
+        {
+            var subject = GetSubject<C2>();
+
+            var queryable = subject.Select(x => new Results { Result = x.Es == value });
+
+            AssertProjection<C2, Results>(queryable, expectedProjection);
+        }
+
+        [Theory]
+        [InlineData(E.A, "{ \"Result\" : { \"$eq\" : [ \"$Es\", \"A\" ] }, \"_id\" : 0 }")]
+        [InlineData(E.B, "{ \"Result\" : { \"$eq\" : [ \"$Es\", \"B\" ] }, \"_id\" : 0 }")]
+        public void Where_operator_equal_should_render_enum_as_string_when_configured_with_class_map(E value, string expectedProjection)
+        {
+            var subject = GetSubject<C3>();
+
+            var queryable = subject.Select(x => new Results { Result = x.Es == value });
+
+            AssertProjection<C3, Results>(queryable, expectedProjection);
+        }
+
+        [Theory]
+        [InlineData(E.A, "{ \"Result\" : { \"$and\" : [{ \"$eq\" : [ \"$Ei\", 0 ] }, { \"$eq\" : [ \"$Es\", \"A\" ] }]}, \"_id\" : 0 }")]
+        [InlineData(E.B, "{ \"Result\" : { \"$and\" : [{ \"$eq\" : [ \"$Ei\", 1 ] }, { \"$eq\" : [ \"$Es\", \"B\" ] }]}, \"_id\" : 0 }")]
+        public void Where_operator_equal_should_render_string_enum_as_string_and_int32_enum_as_int32(E value, string expectedProjection)
+        {
+            var subject = GetSubject<C2>();
+
+            var queryable = subject.Select(x => new Results { Result = x.Ei == value && x.Es == value });
+
+            AssertProjection<C2, Results>(queryable, expectedProjection);
+        }
+
+        // private methods
+        private static void AssertProjection<TDocument, TOutput>(IQueryable<TOutput> queryable, string expectedProjection)
```

Comment:
This is an example of an un-necessary change to `static`.

In theory you should now move this to the static region... but then if you change your mind you have to move it back here.

Since this method is only called from instance methods it should be an instance method also.

Open to feedback. I won't hold up an LGTM just because this method was changed to static.

Reply:
Removed static specifiers.